src/common/mi-lttng-3.0.xsd

```diff
@@ -560,6 +562,32 @@ THE SOFTWARE.
 		</xs:sequence>
 	</xs:complexType>
 
+	<!-- Maps to the rotate command -->
+	<xs:complexType name="rotation_type">
+		<xs:all>
+			<xs:element name="session_name" type="tns:name_type" minOccurs="0" />
+			<xs:element name="status" type="xs:string" />
+			<xs:element name="path" type="xs:string" minOccurs="0" />
+		</xs:all>
+	</xs:complexType>
+
+	<!-- Maps to an array of rotations -->
+	<xs:complexType name="rotations_type">
+		<xs:sequence>
+			<xs:element name="rotation" type="tns:rotation_type" minOccurs="0" maxOccurs="unbounded" />
+		</xs:sequence>
+	</xs:complexType>
+
+	<!-- Maps to the enable-rotation command -->
+	<xs:complexType name="rotation_setup_type">
```

Comment:
Again, for this, my comment about the API could also apply here (including the _schedule_ naming here, etc.).